Assets/Plugins/iOS/mParticleUnity.m

```diff
@@ -530,16 +530,16 @@ + (MPIdentityApiRequest *) MPIdentityApiRequest:(NSDictionary *)identityRequestD
     if ([[identityRequestDict allKeys] containsObject:@"UserIdentities"]) {
         NSDictionary *identities = identityRequestDict[@"UserIdentities"];
         for (id key in identities) {
-            [identityRequest setUserIdentity:identities[key] identityType:(MPUserIdentity)[key integerValue]];
+            [identityRequest setIdentity:identities[key] identityType:(MPUserIdentity)[key integerValue]];
         }
     }
     if ([[identityRequestDict allKeys]containsObject:@"UserAliasUUID"]) {
-        identityRequest.onUserAlias = ^(MParticleUser * _Nonnull previousUser, MParticleUser * _Nonnull newUser) {
+        /*identityRequest.onUserAlias = ^(MParticleUser * _Nonnull previousUser, MParticleUser * _Nonnull newUser) {
             NSDictionary<NSString *, NSString *> *aliasDictionary = @{@"CallbackUuid": identityRequestDict[@"UserAliasUUID"],
                                                                       @"PreviousMpid": [previousUser.userId stringValue],
                                                                       @"NewMpid": [newUser.userId stringValue]};
             UnitySendMessage("MParticle", "OnUserAlias", toChar(jsonWithDictionary(aliasDictionary)));
-        };
+        };*/
```

Comment:
This is the thing I could find what the change in iOS SDK that caused this to break so I just commented it out. The error I was receiving was

	Property 'onUserAlias' not found on object of type 'MPIdentityApiRequest *'

Reply:
@MrTravisB this change it fine - we can delete all of this aliasing code, and it should be replaced by a different API, but OK to comment out for your purpose.